src/Core/Utilities/LoggerFactoryExtensions.cs

```diff
@@ -31,13 +31,16 @@ public static class LoggerFactoryExtensions
     public static ILoggingBuilder AddSerilog(
         this ILoggingBuilder builder,
         WebHostBuilderContext context,
-        Func<LogEvent, bool> filter = null)
+        Func<LogEvent, IGlobalSettings, bool> filter = null)
     {
         if (context.HostingEnvironment.IsDevelopment())
         {
             return builder;
         }
 
+        var globalSettings = new GlobalSettings();
```

Comment:
I know you just moved this further up but we bind `GlobalSettings` twice during startup (here and `AddGlobalSettingsServices()`)? I'd love a tech debt ticket to clean that up.